### DISCUSSION TITLE
Support for android v6

### DISCUSSION THREAD

**Discussion by John4650-hub** [AndroidIDEOfficial/AndroidIDE#619] [General]
Why isn't android v6 supported?
The AIDE IDE project has support for older versions of android though it's nolonger recieving any updates. I wanted to try out this ide but found that my current android version(v6.0.1) is not supported. Do you guys have any plans to include support  for older versions.

**Comment by itsaky** [comment]
I'm sorry, there are no plans to add support for Android 6 devices. You need at least Android 8 to use AndroidIDE.
